trade-finance-manager-ui/server/api.js

```diff
@@ -839,6 +839,22 @@ const downloadUtilisationReport = async (userToken, _id) => {
   };
 };
 
+/**
+ * @param {import('./controllers/utilisation-reports').UpdateReportStatusPayload} updateReportStatusPayload - payload to send in request body
+ * @param {string} userToken - token to validate session
+ * @returns {Promise<import('axios').AxiosResponse>}
```

Comment:
I don't like this return type, but I'm not sure there is a better option. I'm pretty sure nothing specific gets returned in `response.data` as tfm-api just calls `res.sendStatus(204)`. Open to suggestions for better return types here